### ISSUE TITLE
[BIR] Intersection type's functions are not available as top level functions

### ISSUE THREAD

**Issue by pcnfernando** [ballerina-platform/ballerina-lang#30051]
**Description:**
$subject

**Steps to reproduce:**
```ballerina
public type Frame readonly & object {
   public function toString() returns string;
};
```

**Affected Versions:**
SL Alpha 5

**OS, DB, other environment details and versions:**

**Related Issues (optional):**

**Suggested Labels (optional):**

**Suggested Assignees (optional):**

**Comment by gimantha** [comment]
We can close this as we cannot reprduce this in beta1

**Comment by pcnfernando** [comment]
Yes seems the PR was not linked. Linked it now. Fixing this would make the workaround obsolete
